docs/modules/player-lifecycle/heartbeating.md

```diff
@@ -17,16 +17,16 @@ A server sends a heartbeat request at regular intervals to the clients. If the s
 
 ### Initialisation
 
-On the server-worker, the [`PlayerHeartbeatInitializationSystem`](https://github.com/spatialos/gdk-for-unity/blob/master/workers/unity/Packages/com.improbable.gdk.playerlifecycle/Systems/PlayerHeartbeat/PlayerHeartbeatInitializationSystem.cs) checks for ECS entities in the worker's view that have a `PlayerHeartbeatClient` component, but do not yet have the `HeartbeatData` component. For all these entities, a `HeartbeatData` component is added to the ECS entity at the end of the current frame.
+On the server-worker, the [`PlayerHeartbeatInitializationSystem`](https://github.com/spatialos/gdk-for-unity/blob/master/workers/unity/Packages/com.improbable.gdk.playerlifecycle/Systems/PlayerHeartbeat/PlayerHeartbeatInitializationSystem.cs) checks for ECS entities in the worker's view that have a `PlayerHeartbeatClient` component, but do not yet have the [`HeartbeatData`]({{urlRoot}}/api/player-lifecycle/heartbeat-data) component. For all these entities, a [`HeartbeatData`]({{urlRoot}}/api/player-lifecycle/heartbeat-data) component is added to the ECS entity at the end of the current frame.
```

Comment:
Why aren't we linking to the system as well?

Reply:
missed it, gonna fix